control-plane-operator/controllers/hostedcontrolplane/kas/deployment.go

```diff
@@ -429,10 +429,22 @@ func buildKASContainerMain(image string, port int32, noProxyCIDRs []string, hcp
 		c.Command = []string{
 			"hyperkube",
 		}
+
+		// default to level of 2
+		kasVerbosityLevel := "--v=2"
+		if hcp.Annotations[hyperv1.KubeAPIServerVerbosityLevelAnnotation] != "" {
+			kasAnnotationValue := hcp.Annotations[hyperv1.KubeAPIServerVerbosityLevelAnnotation]
+			_, err := strconv.Atoi(kasAnnotationValue)
+			// ensure integer value in annotation
+			if err == nil {
```

Comment:
Not sure what our stance is for the other annotations-as-APIs but exposing the error, when one exists, into status somewhere would help users understand why their attempt at influencing the log level is being ignored.

Reply:
@stevekuznetsov a comment in the Jira indicates that we probably need a logging section in the HCP CRD.  This of course would take a fair amount of work.  As a alternative to this PR as it is now and a full HCP CRD update, would it be acceptable to default KAS logging to v3 instead of v2?  Or maybe allow IBM Cloud to just set v3?  We (IBM) rely on v3 log level to help us debug problems.

Reply:
@stevekuznetsov bump on above comment. Do you have any feedback on this? If more changes are required around logging we would prefer to just bump the default verbosity level instead if that is acceptable.